lnbits/core/templates/core/index.html

```diff
@@ -3,7 +3,6 @@
 {% endblock %} {% block page %}
 <div class="row q-col-gutter-md justify-center">
   <div class="col-12 col-md-7 col-lg-6 q-gutter-y-md">
-    {% if lnurl or LNBITS_NEW_ACCOUNTS_ALLOWED %}
```

Comment:
If you remove this and `LNBITS_NEW_ACCOUNTS_ALLOWED` is set to `false`, then this would create an empty box at the top.